src/main/java/org/jenkinsci/plugins/envinject/EnvInjectPrebuildJobProperty.java

```diff
@@ -0,0 +1,201 @@
+package org.jenkinsci.plugins.envinject;
+
+import hudson.Extension;
+import hudson.FilePath;
+import hudson.model.BuildListener;
+import hudson.model.JobProperty;
+import hudson.model.JobPropertyDescriptor;
+import hudson.model.AbstractBuild;
+import hudson.model.Computer;
+import hudson.model.Descriptor;
+import hudson.model.Job;
+import hudson.model.Node;
+
+import java.io.PrintWriter;
+import java.io.StringWriter;
+import java.util.LinkedHashMap;
+import java.util.Map;
+
+import jenkins.model.Jenkins;
+import net.sf.json.JSONObject;
+
+import org.jenkinsci.lib.envinject.EnvInjectLogger;
+import org.jenkinsci.plugins.envinject.service.EnvInjectEnvVars;
+import org.jenkinsci.plugins.envinject.service.EnvInjectVariableGetter;
+import org.kohsuke.stapler.StaplerRequest;
+
+/**
+ * @author Arcadiy Ivanov (arcivanov)
+ * @author Gregory Boissinot
+ */
+public class EnvInjectPrebuildJobProperty<T extends Job<?, ?>> extends JobProperty<T>
+{
+
+    private EnvInjectJobPropertyInfo info = new EnvInjectJobPropertyInfo();
+    private boolean on;
+
+    @SuppressWarnings("unused")
+    public EnvInjectJobPropertyInfo getInfo()
+    {
```

Comment:
It would be great if you follow the plugin's codestyle. Brackets are being placed at the same line as the declaration in the plugin